app/templates/learnergroups.hbs

```diff
@@ -40,11 +46,11 @@
       <div class="resultslist-title">
         <h2>{{title}}</h2>
       </div>
-      {{#if currenCohort}}
+    {{!-- {{#if currentCohort}} --}}
```

Comment:
I commented-out the 'currentCohort' check, so the button would appear for checking... Need to discuss need for this check with @jrjohnson